prov/rxm/src/rxm.h

```diff
@@ -1018,25 +1018,6 @@ void rxm_buf_release(struct rxm_buf_pool *pool, struct rxm_buf *buf)
 	pool->rxm_ep->res_fastlock_release(&pool->lock);
 }
 
-static inline
-struct rxm_buf *rxm_buf_get_ts(struct rxm_buf_pool *pool)
-{
-	struct rxm_buf *buf;
-
-	fastlock_acquire(&pool->lock);
-	buf = util_buf_alloc(pool->pool);
-	fastlock_release(&pool->lock);
-	return buf;
-}
-
-static inline
-void rxm_buf_release_ts(struct rxm_buf_pool *pool, struct rxm_buf *buf)
-{
-	fastlock_acquire(&pool->lock);
-	util_buf_release(pool->pool, buf);
-	fastlock_release(&pool->lock);
```

Comment:
This is removing the serialization provided by `pool->lock` and not `cmap->lock` (as the commit message implies). If we're removing this it seems we should at least replace it with `util_ep->lock` to support `FI_THREAD_SAFE`.

Reply:
Okay, so rxm_buf_get already acquires pool->lock if needed. I thought it was non-thread safe.